Avoid making an empty libode.a, which is not allowed on macOS.
(ar: no archive members specified)
--- source/Makefile.am.orig	2020-09-01 21:34:07.000000000 -0500
+++ source/Makefile.am	2020-09-02 00:28:30.000000000 -0500
@@ -15,7 +15,10 @@
 # see the file, GPLv2.
 #
 
-noinst_LIBRARIES = libgame.a libode.a
+noinst_LIBRARIES = libgame.a
+if !USE_SYSTEM_LIBODE
+noinst_LIBRARIES += libode.a
+endif
 
 #
 # set location of `data1', etc
